src/components/Atoms/Input/index.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useRef, useEffect } from "react";
+import styled from "styled-components";
+import COLOR from "../../../variables/color";
+import TEXT from "../../../variables/texts";
+
+const InputSample = (props) => {
+  const inputRef = useRef(null);
+  useEffect(() => {
+    if (inputRef.current) {
+      inputRef.current.value = props.defaultValue;

```

Comment:
```suggestion
  useEffect(() => {
    if (!inputRef.current) {
      return;
    }
    inputRef.current.value = props.defaultValue;
```
nullだった場合は早期リターンすることで、ネストが浅くなって可読性が上がるのでおすすめです